### ISSUE TITLE
Inverter name changes to cryptic signs, openDTU crashes

### ISSUE THREAD

**Issue by indie89** [tbnobody/OpenDTU#949]
### What happened?

I use openDTU with 3 Inverters on a ESP32 with a NRF+ Module with external antenna and sufficient power supply (>1A), yet no capacitor. I poll the data every 20s (used to be 5s but I already changed this in order to stop the instabilities). WIFI quality is 100%, I use a mesh-repeater with 20MHz located 50cm away from the DTU.
In the background, I use a iobroker that runs the openDTU Adapter using websocket connections (mqtt is turned off in openDTU) with setting temporary limits every 20 seconds. 

All 1-5 days, the DTU crashes. Part of the crash is a change of one (or more) of the inverter's names, so that it includes cryptic signs (see screenshot). Aftwerwards the openDTU Adapter in iobroker then closes and reopens a websocket connection every ten seconds (unfortunately I can't tell which is first: the websocket loop or the change of inverter's name in openDTU). After some time this leads to a crash of the openDTU adapter in iobroker and normally to a state of the DTU itself, that it is not reachable over Wifi anymore and needs to be hardware-reset (power off/on). Until the DTU finally crashes, polling from the inverters doesn't work anymore (or at least in very long sequences of 5-10 minutes).
![image](https://github.com/tbnobody/OpenDTU/assets/130271301/dd36bbbf-a891-493c-8563-0c42407c656e)

I added the logfiles around the time, the "websocket connection loop" in iobroker started (see below, first start of the loop at "11:14:23"). Maybe that helps narrowing down the problem...

### To Reproduce Bug

I cannot reproduce the bug and it doesn't connect to external factors (at least not in an to me obvious manner). Although I had the same problem with a completely different DTU that did not use an external antenna, so I built a new one, but the bug is not eliminated. I also tried it with just 2 inverters instead of 3, but same bug. I tried to set the poll intervall from 5s to 20s. I tried to change "setting limits" from 5-20 seconds. 

In the logfile I see relatively often "Nothing received, resend whole request" but regularly every inverter is updated within 60seconds.

What I ask myself: which event can lead to a change of name of the inverters? Overloaded? Under-Powered?

### Expected Behavior

I would expect, that the DTU works without crashes. It would be very important to me, for I regularly want to set limits to the inverters and later on want to include another inverter as a battery inverter in order to achieve a "0-Einspeisung". And with the current behaviour, I won't be able to trust the system.

Thanks for your help!!

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

v23.5.22

### Relevant log/trace output

`11:14:18.352 > Fetch inverter: 114xyz
11:14:18.956 > TX RealTimeRunData Channel: 61 --> 15 84 52 91 83 80 12 94 20 80 0B 00 64 6C 83 EB 00 00 00 00 00 00 00 00 C7 5C 87 
11:14:19.621 > RX Period End
11:14:19.621 > All missing
11:14:19.621 > Nothing received, resend whole request
11:14:19.621 > TX RealTimeRunData Channel: 75 --> 15 84 52 91 83 80 12 94 20 80 0B 00 64 6C 83 EB 00 00 00 00 00 00 00 00 C7 5C 87 
11:14:20.253 > RX Period End
11:14:20.253 > All missing
11:14:20.253 > Nothing received, resend whole request
11:14:20.253 > TX RealTimeRunData Channel: 3 --> 15 84 52 91 83 80 12 94 20 80 0B 00 64 6C 83 EB 00 00 00 00 00 00 00 00 C7 5C 87 
11:14:20.929 > Interrupt received
11:14:20.950 > RX Channel: 23 --> 95 84 52 91 83 84 52 91 83 01 00 01 01 4B 00 4C 00 FB 01 4E 00 50 01 09 00 01 7E 
11:14:20.974 > Interrupt received
11:14:20.993 > RX Channel: 23 --> 95 84 52 91 83 84 52 91 83 02 0D CC 00 01 30 D1 00 64 00 8B 09 0A 13 86 01 ED 23 
11:14:21.617 > Interrupt received
11:14:21.639 > RX Channel: 23 --> 95 84 52 91 83 84 52 91 83 83 00 00 00 15 03 E8 00 F5 00 01 82 6E F0 
11:14:21.661 > RX Period End
11:14:21.661 > Success
11:14:21.681 > TX ActivePowerControl Channel: 23 --> 51 84 50 96 00 80 12 94 20 81 0B 00 00 BE 00 01 84 A0 24 
11:14:21.701 > Interrupt received
11:14:21.720 > RX Channel: 75 --> 15 84 52 91 83 84 50 96 00 80 0B 00 64 6C 83 EB 00 00 00 00 00 00 00 00 C7 5C E3 
11:14:21.738 > Websocket: [/livedata][1] disconnect
11:14:22.388 > Websocket: [/livedata][3] disconnect
11:14:22.388 > Websocket: [/livedata][4] disconnect
11:14:23.004 > Websocket: [/livedata][5] connect
11:14:23.701 > RX Period End
11:14:23.701 > All missing
11:14:23.701 > Nothing received, resend whole request
11:14:23.701 > TX ActivePowerControl Channel: 40 --> 51 84 50 96 00 80 12 94 20 81 0B 00 00 BE 00 01 84 A0 24 
11:14:23.724 > Interrupt received
11:14:23.748 > RX Channel: 75 --> D1 84 50 96 00 84 50 96 00 81 00 00 0B 00 14 07 48 
11:14:24.476 > RX Period End
11:14:24.476 > Success
11:14:25.102 > TX ActivePowerControl Channel: 61 --> 51 84 52 91 83 80 12 94 20 81 0B 00 00 64 00 01 7F 81 A2 
11:14:25.744 > Interrupt received
11:14:25.768 > RX Channel: 23 --> 51 84 50 96 00 84 52 91 83 81 0B 00 00 BE 00 01 84 A0 C6 
11:14:26.481 > RX Period End
11:14:26.481 > All missing
11:14:26.481 > Nothing received, resend whole request
11:14:26.481 > TX ActivePowerControl Channel: 75 --> 51 84 52 91 83 80 12 94 20 81 0B 00 00 64 00 01 7F 81 A2 
11:14:27.093 > Interrupt received
11:14:27.776 > RX Channel: 23 --> D1 84 52 91 83 84 52 91 83 81 00 00 0B 00 14 07 48 
11:14:28.927 > RX Period End
11:14:28.927 > Success
11:14:29.544 > TX ActivePowerControl Channel: 3 --> 51 83 71 96 07 80 12 94 20 81 0B 00 00 64 00 01 7F 81 05 
11:14:31.046 > RX Period End
11:14:31.046 > All missing
11:14:31.046 > Nothing received, resend whole request
11:14:31.046 > TX ActivePowerControl Channel: 23 --> 51 83 71 96 07 80 12 94 20 81 0B 00 00 64 00 01 7F 81 05 
11:14:31.918 > Websocket: [/livedata][5] disconnect
11:14:32.534 > Websocket: [/livedata][6] connect
11:14:33.147 > RX Period End
11:14:33.147 > All missing
11:14:33.147 > Nothing received, resend whole request`

**Comment by tbnobody** [comment]
Thats a duplicate of #885

**Comment by indie89** [comment]
No, it's unfortunately not...
Same problem (cryptic signs), that ultimately leads to a crash, but no other connected DTU (neither ahoydtu nor opendtu) at all.

**Comment by tbnobody** [comment]
Just look at the original error in #885. the following error (two dtu) had nothing todo with the original post.

**Comment by indie89** [comment]
Hi, sorry, I'm obviously "standing on the Schlauch ;-)". I see the exact same error but the problem that caused the error (a second dtu connected) is not aplicable to my error. Was there another solution to the error than disconnecting the second DTU? If yes, I didn't see it. How can I solve the problem that causes the error in my case? Pls give me a quick hint... thanks :-)

**Comment by homeautomation2022** [comment]
Better connection to the inverters, you have already tried the Poll interval higher, that would also be my only idea if no other DTU is active in parallel.

**Comment by tbnobody** [comment]
> Was there another solution to the error than disconnecting the second DTU? 

The issue is still open... No solution till now...
